### DISCUSSION TITLE
本地部署提示文章（页面）名不正确？

### DISCUSSION THREAD

**Discussion by Tristan0430** [Qexo/Qexo#465] [Q&A] [answered]
我将qexo和hexo部署在本地，但是qexo不能新建文章和页面，提示没有填入标题。
![image](https://github.com/Qexo/Qexo/assets/113957665/29fdbd5a-7804-4abe-ba95-1ba9542f1d22)

**Comment by Tristan0430** [comment]
@am-abudu @kylinholmes @mis1042 @yzl3014

**Reply by kylinholmes** [reply]
你是容器部署的？细节多讲讲？

**Comment by Jeamorg** [comment, marked as answer]
右上角`文章名称`没填
